ansible/roles/atmo-setup-user/tasks/main.yml

```diff
@@ -10,31 +10,69 @@
   with_dict: "{{ SUDOERS_USERS_AND_GROUPS.TO_REMOVE }}"
   tags: sudoers
 
+- name: remove all instances of atmosphere username in sudoers
+  lineinfile: "dest=/etc/sudoers state=absent regexp='^{{ ATMOUSERNAME }}'"
+  failed_when: false
+
 - name: add users and groups to sudoers
   lineinfile: dest=/etc/sudoers backup=yes regexp={{ item.value.reg }} line={{ item.value.line }} state=present
   with_dict: "{{ SUDOERS_USERS_AND_GROUPS.TO_ADD }}"
   tags: sudoers
 
 - name: start setting up /etc/skel/.bashrc
   lineinfile: dest=/etc/skel/.bashrc line='## Atmosphere System'
+  tags:
+    - cyverse-user-setup
```

Comment:
I would instead split these into another tasks file and conditionally include them when someone sets the appropriate variable, ditto below for the tasks tagged "jetstream-user-setup".